save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/material/Icons.kt

```diff
@@ -0,0 +1,9 @@
+/**
+ * External declarations of icons from fontawesome-solid
+ */
+
+package com.saveourtool.save.frontend.externals.material
+
+@JsModule("@material-ui/icons/KeyboardArrowDown")
```

Comment:
Can we use https://github.com/JetBrains/kotlin-wrappers/tree/master/kotlin-mui-icons?

Reply:
Very good suggestion!

Reply:
added dependencies, also I like their frontend, so https://github.com/saveourtool/save-cloud/issues/967